corehq/apps/reports/filters/forms.py

```diff
@@ -45,8 +45,7 @@ def display_lang(self):
         """
             This should return the lang code of the language being used to view this form.
         """
-        # todo make this functional
-        return 'en'
+        return self.request.couch_user.language or 'en'
```

Comment:
and I think you're implying this, but you're sure that `request` is always populated and `couch_user` is always one of those three classes on `request`. as long as that's right go ahead and self merge.